build-tools/scripts/TestApks.targets

```diff
@@ -152,12 +152,17 @@
 
   <Target Name="RunTestApks"
       Condition=" '@(TestApk)' != '' ">
+    <PropertyGroup>
+      <_InstrumentationArguments Condition=" '$(IncludeCategories)' != '' ">include=$(IncludeCategories)</_InstrumentationArguments>
+      <_InstrumentationArguments Condition=" '$(ExcludeCategories)' != '' ">exclude=$(ExcludeCategories)</_InstrumentationArguments>
```

Comment:
This construct means that you can't use *both* `$(IncludeCategories)` *and* `$(ExcludeCategories)` *at the same time*:

    msbuild Xamarin.Android.sln /t:RunApkTests /p:ExcludeCategories=InetAccess /p:IncludeCategories=Whatever

I'm not immediately sure how useful such a thing is, but if we're going to have `$(IncludeCategories)` *at all*, then we should sensibly support both...

I think that this will work sanely:

```xml
<PropertyGroup>
  <_IncludeCats Condition=" '$(IncludeCategories)' != '' ">include=$(IncludeCategories)</_IncludeCats>
  <_ExcludeCats Condition=" '$(ExcludeCategories)' != '' ">exclude=$(ExcludeCategories)</_ExcludeCats >
</PropertyGroup>
...
<RunInstrumentationTests
    InstrumentationArguments="$(_IncludeCats);$(_ExcludeCats)"
    ...
/>
```